#! /bin/sh
patch -p1 -l -f $* < $0
exit $?

commit b05f232613b64f6aea064d0545223082f60de7fb
Author: Andrej Kacian <ticho@claws-mail.org>
Date:   Sat Jul 25 15:05:18 2015 +0200

    Fix drag&drop attaching files with % in filename.
    
    Closes bug #2115.

diff --git a/src/common/utils.c b/src/common/utils.c
index af1bb20..ab08c44 100644
--- a/src/common/utils.c
+++ b/src/common/utils.c
@@ -1470,7 +1470,7 @@ GList *uri_list_extract_filenames(const gchar *uri_list)
 					if (!locale_file)
 						locale_file = g_strdup(file + 5);
 #else
-					locale_file = g_filename_from_uri(file, NULL, NULL);
+					locale_file = g_filename_from_uri(escaped_utf8uri, NULL, NULL);
 #endif
 					result = g_list_append(result, locale_file);
 				}
